brion/plugin/compartmentReportBinary.cpp

```diff
@@ -204,7 +204,7 @@ size_t CompartmentReportBinary::getFrameSize() const
     return _subtarget ? _comps : _header.numCompartments;
 }
 
-floatsPtr CompartmentReportBinary::loadFrame( const float timestamp ) const
```

Comment:
Yep. I don't have the link at hand, but basically the header is for users (const on by-value is pointless) and the impl is for developers (const means the copy param is not modified)